ddtrace/tracer/util.go

```diff
@@ -63,3 +73,22 @@ func parseUint64(str string) (uint64, error) {
 	}
 	return strconv.ParseUint(str, 10, 64)
 }
+
+func FindContainerID() string {
```

Comment:
We can’t have this as part of the public API. It may be better to put it in the root `internal` package